internal/services/streamanalytics/stream_analytics_output_table_resource.go

```diff
@@ -86,6 +87,15 @@ func (r OutputTableResource) Arguments() map[string]*pluginsdk.Schema {
 			Required:     true,
 			ValidateFunc: validation.IntBetween(1, 100),
 		},
+
+		"columns_to_remove": {
+			Type:     pluginsdk.TypeSet,
```

Comment:
Does this need to be a set? Is the ordering not preserved by the API?

Reply:
Updated